lib/tokenizer/tokenize.js

```diff
@@ -367,6 +371,8 @@ function intoTokens(source, externalContext, internalContext, isNested) {
       isBufferEmpty = true;
     } else if (character == Marker.SEMICOLON && level == Level.RULE && propertyToken && isBufferEmpty) {
       // semicolon after bracketed value at rule level, e.g. a{color:rgb(...);<--
+      if(isVariable && !propertyToken[2]) propertyToken.push([Token.PROPERTY_VALUE, ' ', [originalMetadata(metadata, ' ', externalContext)]]);
```

Comment:
2 changes here: 

1) Please move this `if` case up one level instead of nesting it within existing clause, e.g.

```js
} else if (character == Marker.SEMICOLON && level == Level.RULE && propertyToken && isBufferEmpty && isVariable && !propertyToken[2]) {
  // semicolon after empty variable at rule level, e.g. a{--variable: ;<--
  propertyToken.push([Token.PROPERTY_VALUE, ' ', [originalMetadata(metadata, ' ', externalContext)]]);
  isVariable = false;
  propertyToken = null;
  seekingValue = false;
} else if (character == Marker.SEMICOLON && level == Level.RULE && propertyToken && isBufferEmpty) {
  ...
```

2) Please add a space between `if` and `(` so it's `if (`

Reply:
Done..